pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -533,3 +551,44 @@ func (builder *podEntityDTOBuilder) createContainerPodData(pod *api.Pod) *proto.
 	}
 	return podData
 }
+
```

Comment:
Should we guard these changes with a FeatureGate?  I'm concerned about limited testing of topology.kubernetes.io/zone and topology.kubernetes.io/region on GKE and AKS and the possible need turn off the feature for certain clusters.

Reply:
@pankajbatra1   thanks for reviewing this code change.  according to K8s official document [https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone](https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone), these 2 labels should be trusted, and my code change only works when the pod has a PV attached.  But for sure, I could add a new feature gate for safety. does this looks good to you?
`HonorRegionZoneLabels: {Default: false, PreRelease: featuregate.Alpha}`

Reply:
@pankajbatra1 , I've updated my code by adding a feature gate for this functionality.  Its default value is `false`, if the user want to enable it, the user has to  add the following to the kubeturbo configmap, I tried it out, it works for me.
```
    "featureGates": {
        "HonorRegionZoneLabels": true
    }
```